pkg/corerp/renderers/aci/renderer.go

```diff
@@ -0,0 +1,142 @@
+package aci
+
+import (
+	"context"
+
+	v1 "github.com/radius-project/radius/pkg/armrpc/api/v1"
+	"github.com/radius-project/radius/pkg/corerp/datamodel"
+	"github.com/radius-project/radius/pkg/corerp/renderers"
+	"github.com/radius-project/radius/pkg/resourcemodel"
+	rpv1 "github.com/radius-project/radius/pkg/rp/v1"
+	"github.com/radius-project/radius/pkg/to"
+	"github.com/radius-project/radius/pkg/ucp/resources"
+
+	cs2client "github.com/radius-project/azure-cs2/client/v20230515preview"
+)
+
+const (
+	aciLocation = "West US 3"
```

Comment:
xD

Reply:
This is a POC :) looks like ACI team is frequently deploying the changes and WEST US3 has a latest release all the time. So I hardcoded it for this POC.